doc/source/user_guide/reshaping.rst

```diff
@@ -296,6 +296,25 @@ For instance,
    cheese.melt(id_vars=['first', 'last'])
    cheese.melt(id_vars=['first', 'last'], var_name='quantity')
 
+When transforming a DataFrame using :func:`~pandas.melt`, original the index values can be kept around by setting the ``ignore_index`` parameter to ``False``. This will however duplicate the index values.
+
+.. ipython:: python
+
+   index = pd.MultiIndex.from_tuples([('person', 'first'), ('person', 'second')])
```

Comment:
this example is a bit confusing with first and second since first and last are columns and used as id_vars latter in the example. maybe just 1 and 2 or A and B?